docs/compute-element/install-htcondor-ce.md

```diff
@@ -29,8 +29,11 @@ Before starting the installation process, consider the following points, consult
     `gratia`
     You will also need to create Unix accounts for each collaboration that you wish to support.
     See details in the ['Configuring authentication' section below](#configuring-authentication).
--   **SSL certificate:** The HTCondor-CE service uses a host certificate at `/etc/grid-security/hostcert.pem` and an
-    accompanying key at `/etc/grid-security/hostkey.pem`
+-   **SSL certificate:** The HTCondor-CE service uses a host certificate named
+    `hostcert.pem` and an accompanying key named `hostkey.pem`.
+    -  If using a Let's Encrypt cert, place the cert and key into `/etc/pki/tls`
+    -  If using an IGTF cert, place these under `/etc/grid-security`
```

Comment:
We should mention the exact paths, `/etc/grid-security/host{cert,key}.pem` (but spelled out)

Reply:
Ok, updated.